pkg/apis/kubeone/v1beta1/types.go

```diff
@@ -63,8 +63,13 @@ type KubeOneCluster struct {
 	AssetConfiguration AssetConfiguration `json:"assetConfiguration,omitempty"`
 	// RegistryConfiguration configures how Docker images are pulled from an image registry
 	RegistryConfiguration *RegistryConfiguration `json:"registryConfiguration,omitempty"`
+	// KubeletConfiguration configures the kubelet
+	KubeletConfiguration KubeletConfiguration `json:"kubeletConfiguration,omitempty"`
 }
 
+// Kubele
+type KubeletConfiguration struct{}
```

Comment:
I don't think we need this at all. There's another way we can handle this.

Reply:
done